### PR TITLE
Update CI for the 1.14 release.

### PR DESCRIPTION
**Pull request by amwat**
This PR removes 1.10 jobs and configures 1.14 jobs as per the test-infra handbook
https://github.com/kubernetes/sig-release/tree/master/release-team/role-handbooks/test-infra#create-cipresubmit-jobs-for-the-new-release

/assign @spiffxp 
/sig release
/cc @BenTheElder @krzyzacy @imkin @taragu @munnerz

### CODE REVIEW

**Comment by imkin** [comment]
This was a two step process. Remove 1.10 and then add 1.14 Has something changed?

File: scenarios/kubernetes_verify.py [hunk 1]
```diff
@@ -27,12 +27,12 @@
...
-    '1.11': '1.11-v20190125-cc5d6ecff3',
-    '1.12': '1.12-v20190125-cc5d6ecff3',
-    '1.13': '1.13-v20190125-cc5d6ecff3',
+    '1.11': '1.11-v20190207-b40871ffa',
+    '1.12': '1.12-v20190207-b40871ffa',
+    '1.13': '1.13-v20190207-b40871ffa',
+    '1.14': '1.14-v20190207-b40871ffa',
     # this is master, feature branches...
-    'default': '1.13-v20190125-cc5d6ecff3',
+    'default': '1.14-v20190207-b40871ffa',
```
Comment (krzyzacy) [inline scenarios/kubernetes_verify.py]: we've migrated the verify & integration job to podutils for master - I think it should work for 1.14 as well :-)

**Comment by amwat** [comment]
@imkin upon further clarification during the release meeting, that process was just to ease reviewing smaller diffs. Now that we are automating this stuff, I would go for doing them together since then we don't have to deal with weird edge cases like not having stable3 jobs for one week .

File: config/jobs/kubernetes/kops/kops-config.yaml [hunk 2]
```diff
@@ -94,10 +94,49 @@ presubmits:
         resources:
           requests:
             memory: "6Gi"
-
+  # manual-release-bump-required
+  - name: pull-kops-e2e-kubernetes-aws-beta
```
Comment (krzyzacy) [inline config/jobs/kubernetes/kops/kops-config.yaml]: /shrug - because we have never forked -beta for kops-aws presubmit..

Comment (krzyzacy) [reply config/jobs/kubernetes/kops/kops-config.yaml]: ummmm... this is kubernetes/kops, which... probably shouldn't better

**Comment by krzyzacy** [comment]
(fyi I'm also bumping in https://github.com/kubernetes/test-infra/pull/11264)

File: scenarios/kubernetes_verify.py (hunk 1 above)
Comment (amwat) [reply scenarios/kubernetes_verify.py]: done

File: config/jobs/kubernetes/kops/kops-config.yaml (hunk 2 above)
Comment (amwat) [reply config/jobs/kubernetes/kops/kops-config.yaml]: done

**Comment by krzyzacy** [comment]
(note that https://testgrid.k8s.io/sig-testing-canaries#node-canary is borked mysteriously around noon and not sure if it caused by one of our two images or something... I'm taking back my bump for now)

**Comment by krzyzacy** [comment]
https://github.com/kubernetes/test-infra/pull/11203/commits/fdfa7b47fa7ea35e5793d3ff8ee26b9ad7082d58#diff-89e960ed48e5bafebe5383697c9f724dR166

the beta-verify job need to point to 1.14 and fork from master?

**Comment by krzyzacy** [comment]
and maybe rebump with a new bootstrap?

**Comment by amwat** [comment]
done bumped with bootstrap. Also, the verify job is already forked per branch? I don't think we add a beta name to it since it's a presubmit.

**Comment by krzyzacy** [comment]
oh it didn't give me the line number... https://github.com/kubernetes/test-infra/blob/153655dc3895bc700bea778103a7ca058e3141b1/config/jobs/kubernetes/sig-testing/verify.yaml#L166 the CI one :-)

**Comment by amwat** [comment]
ahh, good catch. done.

**Comment by krzyzacy** [comment]
eh, oops, I meant to say bump bootstrap tag in https://github.com/kubernetes/test-infra/blob/master/images/kubekins-e2e/Dockerfile#L18 and rebuild kubekins :-)

otherwise lgtm :-)

File: config/jobs/kubernetes/sig-testing/bazel-build-test.yaml [hunk 3]
```diff
@@ -31,18 +31,18 @@ presubmits:
...
-    - release-1.13 # per-release job
+      - release-1.14 # per-release job
     labels:
       preset-service-account: "true"
       preset-bazel-scratch-dir: "true"
       preset-bazel-remote-cache-enabled: "true"
     spec:
       containers:
-      - image: gcr.io/k8s-testimages/kubekins-e2e:v20190211-76e197f51-1.13
+      - image: gcr.io/k8s-testimages/kubekins-e2e:v20190213-f6a16b59b-1.13
```
Comment (krzyzacy) [inline config/jobs/kubernetes/sig-testing/bazel-build-test.yaml]: (the image tag was wrong here, fixing in #11370)

File: config/jobs/kubernetes/sig-gcp/gce-conformance.yaml [hunk 4]
```diff
@@ -103,11 +82,10 @@ periodics:
       - --provider=gce
       - --test_args=--ginkgo.focus=\[Conformance\] --ginkgo.skip=Alpha|\[(Disruptive|Feature:[^\]]+|Flaky)\]
       - --timeout=200m
-      image: gcr.io/k8s-testimages/kubekins-e2e:v20190211-76e197f51-1.13
+      image: gcr.io/k8s-testimages/kubekins-e2e:v20190213-f6a16b59b-1.13
 
-# manual-release-bump-required (add a job for the new version and delete the job for the oldest version)
 - interval: 6h
-  name: ci-kubernetes-gce-conformance-stable-1-10
+  name: ci-kubernetes-gce-conformance-latest-1-14
```
Comment (krzyzacy) [inline config/jobs/kubernetes/sig-gcp/gce-conformance.yaml]: (this one is misconfigured ^^)

**Comment by amwat** [comment]
/close 
resolved by https://github.com/kubernetes/test-infra/pull/11370

**Pull request closed without merging**